### DISCUSSION TITLE
Migrate from existing FleetDM to Security Onion Fleet?

### DISCUSSION THREAD

**Discussion by adamphetamine** [Security-Onion-Solutions/securityonion#8565] [Unsupported Versions] [answered]
Hi,
I've installed Security Onion today and I'm very impressed- a lot of work has gone into it- thank you!
I brought this discussion over from Slack as directed. 

I have an existing, working version of Fleet as a standalone install.

Issue- is there an easy way to make the Fleet hosts switch servers?

Initially I figured this wouldn't be hard but as noted in Slack, SO Fleet uses Kolide Launcher and FleetDM uses Orbit, so I may have to remove then re-install the hosts. They all have MDM so it won't kill me to do.

This isn't the worst thing ever, but it would cut into my playtime... so any ideas would be welcome!

Cheers

**Comment by defensivedepth** [comment]
How many hosts are there? And to be clear, they are running Orbit?

**Comment by adamphetamine** [comment, marked as answer]
around 90 hosts, yes running Orbit

**Reply by defensivedepth** [reply]
So if they are running Orbit than there really is no way around it - you will have to do a remove & reinstall with the SO osquery installer. 

I would suggest initially running SO + FleetDM for a bit with a small set of endpoints to test it out and make sure you see how it all works together.
